ges/Example_GES/src/main.cpp

```diff
@@ -0,0 +1,41 @@
+#include <mbed.h>
+#include "DBS_pindefs.h"
+#include "objectlist.h"
+#include "Ethercat.h"
+
+#define WAIT_TIME (2) // seconds
```

Comment:
Why is this time in seconds instead of ms?

Reply:
On a user side, seconds are more intuitive than milliseconds.